### PR TITLE
Add first-time contributor action

### PR DESCRIPTION
**Pull request by delucis**
#### What kind of changes does this PR include?

- Something else!

#### Description

- This PR adds a GitHub action, which comments on PRs from first-time contributors to welcome them.
- Comments with a message based on our current “Next Steps” section in the PR template, which we’re removing in #4946 
- Tested in a test repo during Talking & Doc’ing! Comments look something like this:
Message from a bot account welcoming first-time contributor

### CODE REVIEW

**Review by ElianCodes — APPROVED** [review]
I love it! Merge away!

File: .github/workflows/welcome-bot.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+permissions:
+  pull-requests: write
+
+jobs:
+  welcome:
+    name: Welcome First Time Contributors
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/first-interaction@main
```
Comment (HiDeoo) [inline .github/workflows/welcome-bot.yml:17]:
```suggestion
      - uses: actions/first-interaction@v1
```

**Comment by sarah11918** [comment]
Thanks, everyone, for building this during Talking and Doc'ing!

**Review by TheOtterlord — APPROVED** [review]
Another great workflow in our growing robot army :robot:

**Comment by BryceRussell** [comment]
Looks great!

**Comment by kevinzunigacuellar** [comment]
LGTM!

**Comment by nermalcat69** [comment]
Awesome!

**Review by Genteure — APPROVED** [review]
Awesome welcome message! 👍

**Review by HiDeoo — APPROVED** [review]

**Comment by OliverSpeir** [comment]
👍

**Pull request merged**